doc/classes/ModStorage.adoc

```diff
@@ -0,0 +1,57 @@
+= ModStorage
+
+include::../include/config.adoc[]
+include::../include/types.adoc[]
+:description: Per-mod persistent key-value store
+:keywords: meta, data, metadata, mod, storage, key, value, persistence
+
+ModStorage is a per-world, per-mod persistent string key-value store implementing all methods of MetaData.
+
+The granularity of the persisted snapshots is determined by the `map_save_interval` setting.
+
+WARNING: Incapable of saving raw binary data due to JSON restrictions.
+
+NOTE: It is usually more efficient to leverage the key-value store
+than to store fully serialized data structures; fully serializing the data takes linear time
+in the size of the data whereas updating the key-value store only takes linear time
+in the size of the delta with the new SQLite3 backend. 
+
```

Comment:
perhaps sql vs json backends should be noted somewhere, as well as json taking a bit to save, so if the world crashes, the data might not be saved. additionally, seems that its common people ask which backends does mt support, so a disclaimer that the other db formats supported in maps, etc, is not support at this time

Reply:
> perhaps sql vs json backends should be noted somewhere

good point